src/DogStatsd.php

```diff
@@ -199,8 +201,8 @@ private function isTrue($value)
 
     private function isOriginDetectionEnabled($config)
     {
-        if ((isset($config["origin_detection"]) && !$config["origin_detection"])) {
-            return false;
+        if (isset($config["origin_detection"])) {
+            return $config["origin_detection"];
```

Comment:
I'm a bit rusty on PHP, but would that be worth doing a boolean test?
```
return $config["origin_detection"] == true;
```
or is this array access enough for what we want?

Reply:
This is testing if the value is truthy, rather than strict true, so you could configure `'origin_detection' => 42` and it would be true. From my understanding it is more idiomatic PHP to accept truthy values rather than expect a strict true to be passed in this situation.